internal/telemetry/event.go

```diff
@@ -290,6 +290,40 @@ func withOrgID(cmd CmdFlags, c OrgIDGetter) EventOpt {
 	}
 }
 
+type OutputGetter interface {
+	Output() string
+}
+
+func withOutput(cmd CmdFlags, c OutputGetter) EventOpt {
+	return func(event Event) {
+		if cmd.Flags().Changed(flag.Output) {
+			v, _ := cmd.Flags().GetString(flag.Output)
+			event.Properties["output"] = output(v)
+			return
+		}
+		event.Properties["output"] = output(c.Output())
```

Comment:
Is is safe to send the entire value of --output to our telemetry server?
Templates could contain PII/PHI info.

Reply:
I'm not sending the entire value, only if you see the implementation of `output` it'll send one of ("plaintext", "json", "json-path","go-template","go-template-file")

Reply:
My bad, I've missed that.

Most of the values you're defining below are a copy of what's defined in `output_opts.go`, is that correct?
Why not reuse those values and arrays?

Reply:
you shouldn't export an slice because it can be reassigned  so I decided to follow 
>[A little copying is better than a little dependency.](https://www.youtube.com/watch?v=PAAkCSZUG1c&t=9m28s)